frontend/src/router/index.js

```diff
@@ -149,6 +153,10 @@ const routes = [
         props: true,
       },
     ],
+    meta: {
+      title: "Les mesures phares",
+    },
+    mcGroup: "LAW",
```

Comment:
Qu'est-ce que ça vaut dire le prefix `mc` ?

Reply:
'ma cantine' in order to differentiate it from vue router API, but should think of a better thing since that wasn't clear...